app/validators/status_length_validator.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class StatusLengthValidator < ActiveModel::Validator
-  MAX_CHARS = 500
+  MAX_CHARS = (ENV['MAX_POST_CHARS'] || 500).to_i
```

Comment:
```suggestion
  MAX_CHARS = ENV.fetch('MAX_POST_CHARS', 500).to_i
```

Reply:
We may want to eventually move this into one of the (various open PRs, all pending...) `config_for` areas.